packages/utils/src/format_utils.ts

```diff
@@ -1,7 +1,9 @@
-import { BigNumber } from '@0xproject/utils';
```

Comment:
Let's please not move this to the `utils` package. I am very unsatisfied with these methods that were largely adapted from Web3.js. Let's keep these internal to Web3Wrapper for now.